backends/graphics/opengl/opengl-graphics.cpp

```diff
@@ -1109,9 +1100,14 @@ Surface *OpenGLGraphicsManager::createSurface(const Graphics::PixelFormat &forma
 		// OpenGL ES does not support a texture format usable for RGB555.
 		// Since SCUMM uses this pixel format for some games (and there is no
 		// hope for this to change anytime soon) we use pixel format
-		// conversion to a supported texture format. However, this is a one
-		// time exception.
+		// conversion to a supported texture format.
 		return new TextureRGB555();
+#ifdef SCUMM_LITTLE_ENDIAN
+	} else if (isGLESContext() && format == Graphics::PixelFormat(4, 8, 8, 8, 8, 24, 16, 8, 0)) { // RGBA8888
+#else
+	} else if (isGLESContext() && format == Graphics::PixelFormat(4, 8, 8, 8, 8, 0, 8, 16, 24)) { // ABGR8888
+#endif
+		return new TextureRGBA8888Swap();
```

Comment:
Do we need to also systematically use the TextureRGBA8888Swap if either `USE_FORCED_GLES` or `USE_FORCED_GLES2` is defined (i.e. have the same conditions that what we had before when checking if those pixel formats were supported)?

Reply:
This code is already disabled when built with `USE_FORCED_GL`, so I don't think any additional checks are necessary.